packages/fast-components-styles-msft/src/listbox/index.ts

```diff
@@ -0,0 +1,25 @@
+import designSystemDefaults, {
+    DesignSystem,
+    withDesignSystemDefaults,
+} from "../design-system";
+import { ComponentStyles, ComponentStyleSheet } from "@microsoft/fast-jss-manager";
+import { ListboxClassNameContract } from "@microsoft/fast-components-class-name-contracts-base";
+import { foregroundLarge } from "../utilities/colors";
+import { elevation, ElevationMultiplier } from "../utilities/elevation";
+import { toPx } from "@microsoft/fast-jss-utilities";
```

Comment:
no need to import this as it is not needed, same with elevation and foreground from what I can see.

Reply:
and topx...deleted